app/controllers/nps_surveys/api_internal/responses_controller.rb

```diff
@@ -7,11 +7,16 @@ def create
 
         return render json: ResponsePresenter.new(response).to_json if response.valid?
 
-        raise AppException::UnprocessableEntity, response
+        render json: JSON.generate(
```

Comment:
do we need `JSON.generate()` here? we should be able to pass a Hash directly to `json:`